Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -9,14 +9,19 @@ The Mixed Reality Toolkit (MRTK) is a cross-platform toolkit for building Mixed
 To get started with the Mixed Reality Toolkit you will need:
 
 * [Visual Studio 2017](http://dev.windows.com/downloads)
-* [Unity 2018.3.x](https://unity3d.com/get-unity/download/archive)
+* [Unity 2018.3.7](https://unity3d.com/get-unity/download/archive)
```

Comment:
Was the .7 part intentional here?

Reply:
yes. We have found issue with 2018.3.14 that blocks deployment at two hack event, and seems to have no issue with 2018.3.7.